.scss-lint.yml

```diff
@@ -83,7 +83,7 @@ linters:
     extra_properties: []
   QualifyingElement:
     enabled: true
-    allow_element_with_attribute: false
+    allow_element_with_attribute: true
```

Comment:
Does this need to be disabled everywhere? Else you can disable just for one file/selector https://github.com/AusDTO/gov-au-ui-kit/blob/c298510bc21550be59a1dce5ef9a746727d1898b/assets/sass/_block-elements.scss#L65